packages/spatial/tsconfig.json

```diff
@@ -41,5 +41,5 @@
     "../ecs/**/*",
     "./**/*.ts",
     "./**/*.tsx"
-  ]
```

Comment:
please check this is correct as per our indentation

Reply:
implemented